apps/api/src/app.ts

```diff
@@ -17,6 +17,7 @@ import { dashAnalyticsRouter } from '@api/routes/dash';
 import { mediaRouter } from '@api/routes/media';
 import { onboardingRouter } from '@api/routes/onboarding';
 import { organizationRouter } from '@api/routes/organization';
+import { communityRouter } from './routes/community';
```

Comment:
This is not how other routers are imported here

Reply:
fixed